base/applications/utilman/umandlg/lang/uk-UA.rc

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
BOM?

Reply:
Some of the translation files erroneously had encoding as UTF-8 with BOM so I changed them to UTF-8 instead, this is intentional. Check the commit description.

Reply:
Ok, thanks!